pyuvdata/tests/test_utils.py

```diff
@@ -11,6 +11,8 @@
 import numpy as np
 from pyuvdata.data import DATA_PATH
 import pyuvdata.utils as uvutils
+from pyuvdata.miriad import Miriad
```

Comment:
Why do you need this import? It doesn't look like you added anything that uses it...

Reply:
I may have added that in by accident..